package.json

```diff
@@ -3,19 +3,24 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "dev": "next dev",
-    "dev-ssl": "npm-run-all --parallel ssl-proxy dev",
+    "dev": "run-p ssl-proxy dev:insecure",
+    "dev:insecure": "next dev",
     "ssl-proxy": "local-ssl-proxy --source 4000 --target 3000",
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "prisma": "dotenv -c development prisma",
-    "test": "run-s lint testUnit testIntegration",
-    "testUnit": "jest",
-    "testIntegration": "start-server-and-test dev-ssl http://localhost:3000 cypress:headless",
-    "tdd": "jest --watch",
-    "cypress": "cypress open",
-    "cypress:headless": "cypress run"
+    "prisma": "dotenv -c development -- prisma",
+    "prisma:test": "dotenv -c test -- prisma",
+    "prisma:test:migrate": "dotenv -c test -- prisma migrate reset --force",
+    "test": "run-s lint test:unit:run test:e2e:run",
+    "test:unit:run": "jest",
+    "test:unit:dev": "jest --watch",
+    "test:e2e:run": "cross-env NODE_ENV=test start-server-and-test \"run-s docker:up dev\" :3000 \"run-s prisma:test:migrate cy:run docker:down\"",
+    "test:e2e:dev": "cross-env NODE_ENV=test start-server-and-test \"run-s docker:up dev\" :3000 \"run-s prisma:test:migrate cy:dev docker:down\"",
+    "cy:run": "cypress run",
+    "cy:dev": "cypress open",
+    "docker:up": "docker-compose up -d",
+    "docker:down": "docker-compose down -v *"
```

Comment:
This doesn't work on Windows, and instead spits out the help options. There are no mounted volumes as of yet, so I think it should work fine if you drop the `-v *`.

Reply:
If I don't have this the E2E command leaves behind a volume each time you run it. I'll take a closer look tomorrow.

Reply:
Even on Linux, the set of containers is left running. I don't think a volume should be created or mounted, it's not specified in the `docker-compose.yml`

Reply:
This is what happens if I run `>npm run docker:up`, followed by `>npm run docker:down`,  followed by `>npm run docker:up`, without `-v *` (notice there are two volumes). I can confirm this behaviour also occurs on Windows using the command prompt and that it works fine (for me) with `-v *`. What is the specific error you are getting on Windows?

![image](https://user-images.githubusercontent.com/684772/144146370-238566a6-2d98-4f26-86b2-ea5f3b64983a.png)

Reply:
I am using PowerShell ; this is likely why this is happening. It outputs the usage info for `docker-compose down` when it tries to run `docker-compose down -v *`. It leaves the PostgreSQL database running and the volume mounted. I can confirm that the volume is left there if you just run `docker-compose down`.

I will try on command prompt and see what happens...

Reply:
Same thing when using command prompt; doesn't stop the database container or unmount the volume

Reply:
If I remove the asterisk (i.e. run `docker-compose down -v` it stops the container and unmounts the volume

Reply:
Thanks for finding the solution! Fixed